final_task/rss_reader/classes/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class FeedError(Exception):
+    '''
+    Raise when something is wrong with feed
+    Really I don't feel good about this class
```

Comment:
This class is correct. Although it is better to have multiple custom exception classes for each specific error case